docs/device-api.md

```diff
@@ -14,118 +14,122 @@ a call to one of the [API Endpoints](#api-endpoints) will fail with a
-|                            | tx_rate_limiter       |    O     |       O        |      O       |     **R**     |      O       |      O     |
-| `PartialDrive`             | drive_id              |    O     |       O        |    **R**     |       O       |      O       |      O     |
-|                            | path_on_host          |    O     |       O        |    **R**     |       O       |      O       |      O     |
-| `PartialNetworkInterface`  | iface_id              |    O     |       O        |      O       |     **R**     |      O       |      O     |
-|                            | rx_rate_limiter       |    O     |       O        |      O       |     **R**     |      O       |      O     |
-|                            | tx_rate_limiter       |    O     |       O        |      O       |     **R**     |      O       |      O     |
-| `RateLimiter`              | bandwidth             |    O     |       O        |      O       |     **R**     |      O       |      O     |
-|                            | ops                   |    O     |       O        |    **R**     |       O       |      O       |      O     |
-| `TokenBucket`<sup>\*</sup> | one_time_burst        |    O     |       O        |    **R**     |       O       |      O       |      O     |
-|                            | refill_time           |    O     |       O        |    **R**     |       O       |      O       |      O     |
-|                            | size                  |    O     |       O        |    **R**     |       O       |      O       |      O     |
-| `TokenBucket`<sup>\*</sup> | one_time_burst        |    O     |       O        |      O       |     **R**     |      O       |      O     |
-|                            | refill_time           |    O     |       O        |      O       |     **R**     |      O       |      O     |
-|                            | size                  |    O     |       O        |      O       |     **R**     |      O       |      O     |
-| `Vm`                       | state                 |    O     |       O        |      O       |       O       |      O       |      O     |
-| `Vsock`                    | guest_cid             |    O     |       O        |      O       |       O       |    **R**     |      O     |
-|                            | uds_path              |    O     |       O        |      O       |       O       |    **R**     |      O     |
-|                            | vsock_id              |    O     |       O        |      O       |       O       |    **R**     |      O     |
-| `EntropyDevice`            | rate_limiter          |    O     |       O        |      O       |       O       |      O       |    **R**   |
-
-<sup>\*</sup>: The `TokenBucket` can be configured with any combination of
+| Schema                     | Property              | keyboard | serial console | virtio-block | vhost-user-block |  virtio-net   | virtio-vsock | virtio-rng |
```

Comment:
I honestly have no clue what value do we think this table has.

Reply:
agree, just updated them for consistency to illustrate that some of the properties are relevant for one of the device types